.github/workflows/ci.yml

```diff
@@ -5,6 +5,14 @@ on:
     branches:
       - 'main'
   pull_request:
+    # Ignore PRs from release-please.
+    # Might also ignore the rare PR containing only changes to package.json.
+    paths-ignore:
+      - 'changelog.json'
+      - 'CHANGELOG.md'
+      - 'package.json'
+      - 'package-lock.json'
+      - 'VERSION'
```

Comment:
What value are we adding by ignoring this workflow based on the paths? If the goal was to ignore release-please PR, why not use `branches-ignore`?

Reply:
Unfortunately `branches-ignore` refers to the branch being merged into, not the branch of the PR being merged.

Reply:
Rats ok. I still question the value of adding these ignores in these two workflow.

Reply:
Rats ok. I still question the value of adding these ignores in these two workflow.

Reply:
Can we block the action based on the user?

Reply:
```release-please requires a GitHub token to access the GitHub API. You configure this token via the token configuration option. You can use the built-in GITHUB_TOKEN secret, however, note that any resources created by release-please (release tag or release pull request) will not trigger future GitHub actions workflows.```

Is this feasible to stop the action from running?